index.html

```diff
@@ -290,7 +291,42 @@ <h2>About me</h2>
         </div>
       </div>
     </main>
-
+    <footer>
+      <form
+        action="https://formspree.io/f/xdoyydwq"
+        method="POST"
+        class="form-container"
+      >
+        <p>
+          I'm always interested in hearing about new projects, so if you'd like
+          to chat please get in touch.
+        </p>
+        <div class="inputs">
+          <div class="group-input">
+            <div class="fname">
+   
+              <input type="text" placeholder="First Name" />
+            </div>
+            <div class="lname">
+          
+              <input type="text" placeholder="Last Name"/>
+            </div>
+          </div>
+          <div class="email">
+        
+            <input type="email" placeholder="Email address"/>
+          </div>
+          <textarea
+            name="body"
+            id="body"
+            cols="30"
+            rows="10"
+            placeholder="Hello Mariane. Glad to see you |"
+          ></textarea>
```

Comment:
- [ ] The massage input should also have a `maxlength="500"` attribute it is one of the project requirements just like I've mentioned above. Let us work on this to make the project according to the required standards.